docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -5,6 +5,10 @@ You can establish communication between a media control server and a media contr
 
 The main media controller features include:
 
+- Updating and Retrieving Playlist
```

Comment:
Query:
@mijin85cho, I think since this bullet list is not a title, it need not be written in title case (first letter in capital). Rather the words can be written in sentence case (first letter in smaller) as follows:

Can we make:
- 'Updating and Retrieving Playlist' to 'Updating and retrieving playlist'
- 'Setting and Getting Playback Capability' to 'Setting and getting playback capability'
- 'Media Control Metadata Properties' to 'Media Control Metadata Properties'

Please share your thoughts.

Suggestion:
@hsgwon
In line 71, there is an extra colon (:) at the end of the sentence, please remove it.

Reply:
Hello @divyk 
Thanks for good pointing out. As this list is about "main media controller features", each list has link to the related section, but it doesn't need to use same sentence form with the section title. 

I think using suggested form like 'Updating and retrieving playlist' or  'Setting and getting playback capability' is ok.

Reply:
Done